ReactWindows/ReactNative.Shared/Modules/Network/HttpContentHelpers.cs

```diff
@@ -1,27 +1,43 @@
 ﻿using System;
 using System.IO;
 using System.IO.Compression;
+#if WINDOWS_UWP
```

Comment:
>WINDOWS_UWP [](start = 4, length = 11)

Would love hear how @matthargett feels about this and NetworkingModule.  The original approach may have been cleaner after all (I thought the type aliases were going to be the only #ifs).